lib/beacon/config.ex

```diff
@@ -12,6 +12,9 @@ defmodule Beacon.Config do
     * `:data_source` (optional) `t:data_source/0` - a module that implements `Beacon.DataSource.Behaviour`,
       used to provide assigns to your site pages.
 
+    * `:authorization_source` (optional) `t:authorization_source/0` - a module that implements `Beacon.Authorization.Behaviour`,
+      used to provide authorization rules for the admin backend.
```

Comment:
```suggestion
      used to provide authorization rules for the admin backend. Defaults to `Beacon.Authorization.DefaultPolicy` and note this config can't be `nil`.
```

Feel free to change wording here but we have to make it explicit this config can't be `nil` because admin depends on it. It's optional because we provide a default policy but it has to be valid.